crates/cdk/src/cdk_lightning/mod.rs

```diff
@@ -121,16 +121,18 @@ pub struct PaymentQuoteResponse {
 }
 
 /// Ln backend settings
-#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
why drop `Copy`?

Reply:
Mint and Melt settings have `PaymentMethod` which one variant is a custom string so it can't be `copy` but I think we can remove the custom or make it a static str and should be able to add copy back. Could make that its own pr or add it here I don't mind.